### DISCUSSION TITLE
Bug with comcast.net

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#2185] [Help with using Postal] [answered]
Not sure of anyone else has seen this, but my recent install Postal installation is working fine with just one exception, I cant send any email to comcast - it appears to be an internal postal error - not smtp related.

October 14, 2022 11:59 ERROR

An internal error occurred while sending this message. This message will be retried automatically. If this persists, contact support for assistance.

Mysql2::Error: Data too long for column 'output' at row 1
Support Ref: J-RROX9ZD2WV

It will retry indefinitely, the email addresses in question I know work fine. Looks like an internal mysql error, but not sure how to debug further. Can anyone else send to comcast? All other emails/domains work fine, and we send to over 300,000 subscribers, only Comcast is doing this.

**Comment by ghost** [comment, marked as answer]
Update... Comcast SMTP error was too long for the varchar(512) output column in deliveries table. I had to alter it to varchar(4096) to capture the error. 

I recommend a bugfix where when the smtp status message is greater then 512 chars, just truncated it and insert the 1st 512 chars of message vs trying to insert the whole message which then triggers this error.

In the end it was legit 451 deny that I was able to fix, once I saw the error.

**Reply by willpower232** [reply]
glad you were able to figure it out, are you able to make a migration and a PR? you should be able to reference the other files here https://github.com/postalserver/postal/tree/main/lib/postal/message_db/migrations

**Reply by ghost** [reply]
I created https://github.com/postalserver/postal/pull/2187 - not sure what the process is to get someone to review/merge/release it.

**Reply by willpower232** [reply]
I'm sure they'll appear eventually, at least it is logged clearly as a code change now thank you :pray:
